lib/cli/package.json

```diff
@@ -18,7 +18,8 @@
   "license": "MIT",
   "author": "Storybook Team",
   "bin": {
-    "getstorybook": "./bin/index.js"
+    "getstorybook": "./bin/index.js",
```

Comment:
We could also decide to keep these 2 separate, and keep all backwards compatibility in the getstorybook.js file.

That might be easier to remove?

Reply:
I tried this locally, but when I change the entrypoint for `getstorybook` to something else the command is not detected, not sure what's happening here 🤔 the symlink is present tho